python/whylogs/viz/utils/html_template_utils.py

```diff
@@ -18,9 +18,10 @@ def _get_compiled_template(template_name: str) -> "Callable":
     try:
         from pybars import Compiler  # type: ignore
     except ImportError as e:
+        msg = "Unable to load pybars; install pybars3 to load profile directly from the current session "
         logger.debug(e, exc_info=True)
-        logger.warning("Unable to load pybars; install pybars3 to load profile directly from the current session ")
```

Comment:
can you include the hint:
...or `pip install whylogs[viz]` to install dependencies for this module